baseConfig.json

```diff
@@ -181,6 +181,8 @@
         "no-magic-numbers": "off",
         "no-mixed-operators": "off",
         "array-bracket-spacing": [ "error", "always" ],
+        "array-bracket-newline": "off",
+        "array-element-newline": [ "error", { "multiline": true } ],
```

Comment:
Does this mean that arrays containing non-multiline elements are not allowed to have newlines? This is not clear to me from the eslint rule examples...